### PR TITLE
chore(datagrid): add expanded row visual tests

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
No visual regression tests for datagrid expandable rows.

Issue Number: CDE-1692

## What is the new behavior?
Visual regression tests for datagrid expandable rows are added.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
Follow up on #1288

### CODE REVIEW

File: .storybook/stories/datagrid/datagrid-expandable-row.stories.ts [hunk 1]
```diff
@@ -14,6 +14,12 @@ import { setupStorybook } from '../../helpers/setup-storybook.helpers';
 
 const defaultStory: Story = args => ({
   template: `
+<style>
+.open-tooltip {
+  visibility: visible;
+  opacity: 1;
+}
+</style>
```
Comment (kevinbuhmann) [inline .storybook/stories/datagrid/datagrid-expandable-row.stories.ts:22]:
```suggestion
  <style>
  .open-tooltip {
    visibility: visible;
    opacity: 1;
  }
  </style>
```

missing indentation

File: .storybook/stories/datagrid/datagrid-expandable-row.stories.ts [hunk 2]
```diff
@@ -57,9 +63,14 @@ const defaultStory: Story = args => ({
             solid
           ></cds-icon>
           <span class="tooltip-content"
+            [class.open-tooltip]="openTooltip&&index===0"
```
Comment (kevinbuhmann) [inline .storybook/stories/datagrid/datagrid-expandable-row.stories.ts:66]:
```suggestion
            [class.open-tooltip]="openTooltip && index === 0"
```

missing spaces

**Review by kevinbuhmann — CHANGES_REQUESTED** [review]

File: .storybook/stories/datagrid/datagrid-expandable-row.stories.ts (hunk 2 above)
Comment (valentin-mladenov) [reply .storybook/stories/datagrid/datagrid-expandable-row.stories.ts]: corrected in commit 3c9de6e

File: .storybook/stories/datagrid/datagrid-expandable-row.stories.ts (hunk 1 above)
Comment (valentin-mladenov) [reply .storybook/stories/datagrid/datagrid-expandable-row.stories.ts]: corrected in commit 3c9de6e

File: .storybook/stories/datagrid/datagrid-expandable-row.stories.ts [hunk 3]
```diff
@@ -14,6 +14,12 @@ import { setupStorybook } from '../../helpers/setup-storybook.helpers';
 
 const defaultStory: Story = args => ({
   template: `
+  <style>
+  .open-tooltip {
+    visibility: visible;
+    opacity: 1;
+  }
+  </style>
```
Comment (kevinbuhmann) [inline .storybook/stories/datagrid/datagrid-expandable-row.stories.ts:22]:
```suggestion
  <style>
    .open-tooltip {
      visibility: visible;
      opacity: 1;
    }
  </style>
```

still missing indentation

**Review by kevinbuhmann — APPROVED** [review]

**Review by kevinbuhmann — APPROVED** [review]

**Pull request merged**